templates/book/index.html.twig

```diff
@@ -62,61 +65,83 @@
+                {% if book.serie %}
                 <tr>
                     <th>{{ 'book.serie'|trans }}</th><td>
                         <div>
-                            {{ component('InlineEditBook', {'book':book, 'field':'serie'}) }}
+                            <a href="{{ filter_book_url({'serie': book.serie}) }}" class="text-decoration-none text-secondary">
+                                <i class="bi bi-list-ol"></i>&nbsp;{{ book.serie }}
+                            </a>
+                            {% if book.serie is not null %}
+                            #{{ book.serieIndex }}
+                            {% endif %}
                         </div>
-                        {% if book.serie is not null %}
-                            <div class="w-25 d-inline-block">
-                                {{ component('InlineEditBook', {'book':book, 'field':'serieIndex'}) }}
-                            </div>
-                        {% endif %}
+
 
                     </td>
                 </tr>
+                {% endif %}
+                {% if book.publisher %}
                 <tr>
-                    <th>{{ 'book.publisher'|trans }}</th><td>{{ component('InlineEditBook', {'book':book, 'field':'publisher'}) }}</td>
+                    <th>{{ 'book.publisher'|trans }}</th><td>{{ book.publisher }}</td>
                 </tr>
+                {% endif %}
 
+                {% if book.ageCategory is not null %}
                 <tr>
-                    <th>{{ 'book.age-category'|trans }}</th><td>{{ component('InlineEditBook', {'book':book, 'field':'ageCategory'}) }}</td>
+                    <th>{{ 'book.age-category'|trans }}</th><td>
+                        {% if book.ageCategory is not null %}
+                            {{ enum("App\\Enum\\AgeCategory").getLabel(book.ageCategory)|trans }}
+                        {% else %}
+                            {{ "enum.agecategories.notset"|trans }}
+                        {% endif %}
+                    </td>
                 </tr>
+                {% endif %}
 
+                {% if book.tags %}
                 <tr>
-                    <th>{{ 'book.tags'|trans }}</th><td>{{ component('InlineEditBook', {'book':book, 'field':'tags'}) }}</td>
+                    <th>{{ 'book.tags'|trans }}</th><td>
+                        {% for tag in book.tags %}
+                            <a href="{{ filter_book_url({'tag': tag}) }}" class="text-decoration-none text-secondary">
+                                <i class="bi bi-tag-fill"></i>&nbsp;{{ tag }}
+                            </a>
+                        {% endfor %}
+                    </td>
                 </tr>
+                {% endif %}
+                {% if book.language %}
                 <tr>
-                    <th>{{ 'book.language'|trans }}</th><td>{{ component('InlineEditBook', {'book':book, 'field':'language'}) }}</td>
+                    <th>{{ 'book.language'|trans }}</th><td>{{ book.language }}</td>
```

Comment:
Shouldn't this show the language like "French" instead of "fr" ?

Reply:
#394